### PR TITLE
Add new function update_vm_disk_driver_cache()

### PR DESCRIPTION
**Pull request by fangge1212**
Signed-off-by: Fangge Jin <fjin@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3151,6 +3151,38 @@ def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
...
+        # Update disk driver cache mode
+        driver_dict = disk.driver
+        driver_dict['cache'] = driver_cache
+        disk.driver = driver_dict
+        logging.debug("The new vm disk driver cache is %s", disk.driver['cache'])
+
+        vmxml.devices = devices
+
+        # SYNC VM XML change
+        logging.debug("The new VM XML:\n%s", vmxml.xmltreefile)
```
Comment (kylazhang) [inline virttest/utils_test/libvirt.py]: change vmxml.xmltreefile to vmxml should be the same

**Comment by kylazhang** [comment]
others looks good to me

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (fangge1212) [reply virttest/utils_test/libvirt.py]: updated

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -3151,6 +3151,38 @@ def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
         return (False, log)
 
 
+def update_vm_disk_driver_cache(vm_name, driver_cache="none", disk_index=0):
+    """
+    Update disk driver cache of the VM
+
+    :param driver_cache: new vm disk driver cache mode, default to none if not provided
+    :param disk_index: vm disk index to be updated, the index of first disk is 0.
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py]: miss one parameter doc string

Comment (fangge1212) [reply virttest/utils_test/libvirt.py]: updated

**Pull request merged**